walk.go

```diff
@@ -19,7 +19,7 @@ func ListGoFiles(dirPath string, ignore []string) ([]string, error) {
 			}
 		}
 		// Check if the file has a ".go" extension
-		if strings.HasSuffix(info.Name(), ".go") && !strings.HasSuffix(info.Name(), "_test.go") {
+		if strings.HasSuffix(info.Name(), ".go") && !strings.HasSuffix(info.Name(), "_test.go") && !strings.HasSuffix(info.Name(), ".gen.go") {
```

Comment:
I don't know how this is happening, since we blacklist the output folder from the walk function, it should never run into a `.gen.go` file, but good addition.